Dockerfile

```diff
@@ -108,6 +108,15 @@ COPY $NODE_PRIVKEY_FILE /home/$BUILD_USER/massa_exec_files/massa-node/config/nod
 COPY $NODE_CONFIG_INITIAL_LEDGER /home/$BUILD_USER/massa_exec_files/massa-node/base_config/initial_ledger.json
 COPY $NODE_CONFIG_INITIAL_ROLLS /home/$BUILD_USER/massa_exec_files/massa-node/base_config/initial_rolls.json
 COPY wait_ts.py /home/$BUILD_USER/massa_exec_files/massa-node
+COPY update_key.py /home/$BUILD_USER/massa_exec_files/massa-node
+COPY requirements_tests.txt /home/$BUILD_USER/massa_exec_files/massa-node
```

Comment:
No need to create another virtual env, let's use the same as for the config.py script

Reply:
We are using the same virtual env, we just need to install dependencies present in the requirements_tests.txt to run the update_key script